types/nightwind/index.d.ts

```diff
@@ -0,0 +1 @@
+// nightwind does only provide content at nightwind/helper
```

Comment:
Please add correct types here:

```js
const x = require("nightwind");
console.log(x);
```

results in:
```
{
  handler: [Function (anonymous)],
  config: { theme: { extend: [Object] } }
}
```

Reply:
Yea, those are tailwind plugin definitions. Since they are sort of internal (users use this, but only in the tailwind config) I thought not including them would be better.

I have attached the nessesary types and definitons